### PR TITLE
Updated dispatch version to 1.16.0-freeze and proton version to 0.34.0

### PR DESCRIPTION
**Pull request by fgiorgetti**

### CODE REVIEW

**Comment by fgiorgetti** [comment]
The idea of this change to `Makefile` is to avoid pushing the latest tag if we are building code freeze.

File: Makefile [hunk 1]
```diff
@@ -5,8 +5,8 @@ DOCKER_ORG=interconnectedcloud
 PWD=$(shell pwd)
 
  # This is the latest version of the Qpid Dispatch Router
-DISPATCH_VERSION=1.15.0
-PROTON_VERSION=0.33.0
+DISPATCH_VERSION=1.16.0-freeze
```
Comment (grs) [inline Makefile:8]: This won't work for the url below.

**Comment by grs** [comment]
Perhaps the tag to be pushed should come from the branch of this repo or something?

File: Makefile (hunk 1 above)
Comment (fgiorgetti) [reply Makefile:8]: Yes, I was testing consuming it from github and it worked, but not working from apache,
I will research a bit more and I will try not to force tags to be pushed into this repo initially.

**Comment by fgiorgetti** [comment]
@grs @ganeshmurthy using github as a source for the tarball instead of apache, it is going through.
Let me know if you guys think this can be merged so we can evaluate the image once it is published.

**Review by grs — APPROVED** [review]

**Pull request merged**